.nvmrc

```diff
@@ -1 +1 @@
-12.14
+12.14.1
```

Comment:
Why pin this so specifically? Generally minor version ranges are not breaking.

Reply:
people who use asdf version manager require a fully-qualified version, and its absence breaks the tool when pulling changes in. Alternative would be for us folks to use the asdf `.tool-versions` file instead of this file, just needing to make sure they stay up to date.